call-notes/call_6.md

```diff
@@ -0,0 +1,160 @@
+
+#### Questions
+
+Video |  [22:59](https://youtu.be/GA5AVg_svj8?t=1379)
+-|-
+
+**Wen Xu:** Let me know if you have questions.
+
+**Mango Team:** Can you hear me?
+
+**Jacob Creech:** Yes, we can hear you.
+
+**Wen Xu:** Yep, yep!
+
+**Mango Team:** Yeah, so actually I am from Mango team and like we are currently implementing [SIMD 47](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0047-syscall-and-sysvar-for-last-restart-slot.md) which is like the last restart slot. So,  I am wondering if this silent repair stage should be considered also a restart slot?
+
+**Wen Xu:** Sorry I haven't checked that document in detail. So I think, correct me if I am wrong, your proposal is to expose the last restarted slot somehow through Ledger right?
+
+**Mango Team:** Yeah, exactly.
+
+**Wen Xu:** Yeah so, here I think it is orthogonal but once the silent repair phase is over and we know where we are restarting from. We could also like connect to your code and expose this information somewhere right?
+
+**Mango Team:** Oh yeah exactly, so like silent repair should be considered as a restart right? Because it may take time to restart the slot silently right? That's my question.
+
+**Wen Xu:** So when it is in the silent repair phase, we don't know for sure it's a restart. They are negotiating but they don't know whether they can decide on the same block. So in that case I don't think we will expose anything. And once that phase is over and we know we are really entering the restart, then we can expose that restart slot. Did that answer the question?
+
+**Mango Team:** Okay! yeah, it makes sense yeah.
+
+**Anonymous:** There shouldn't actually be any changes to that proposal because once we've committed to restarting at the coordinated restart slot there is going to be a hard Fork anyway and we will. So that will update this as far.
+
+**Mango Team:** Okay, okay that's cool.
+
+**Wen Xu:** Yeah. So, that works nicely together. And any other questions?
+
+**Jacob Creech:** Any have a question?
+
+**Validator Operator** Okay sure so as a validator operator, I want to say it is very encouraging to see how much really good thought design seems to have gone into what you are proposing here. I predict that what would happen during another restart event is that there would still be humans that will get alerting and there will be a lot of confusion and talk. So, if we are kind of raising some automated process, just keep in mind that you may want controls there as certainly messaging from the validator to sort of let us be very aware of what's going on so that we can make decisions about. Because if someone believes that there's a bug or a reason that the restart shouldn't proceed allowing something to run away with a whole cluster restart that we want to pause. Just having controls and informational messages to help us understand what's happening are very important. I just want to emphasize that. That's all.
```

Comment:
```suggestion
**Validator Operator** Okay sure so as a validator operator, I want to say it is very encouraging to see how much really good thought design seems to have gone into what you are proposing here. I predict that what would happen during another restart event is that there would still be humans that will get alerting and there will be a lot of confusion and talk. So, if we are kind of racing some automated process, just keep in mind that you may want controls there and certainly messaging from the validator to sort of let us be very aware of what's going on so that we can make decisions about. Because if someone believes that there's a bug or a reason that the restart shouldn't proceed allowing something to run away with a whole cluster restart that we want to pause. Just having controls and informational messages to help us understand what's happening are very important. I just want to emphasize that. That's all.
```